src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -421,11 +453,13 @@ export function selectModalityIcon(appointment) {
   const isInPerson = selectIsInPerson(appointment);
   const isCommunityCare = selectIsCommunityCare(appointment);
 
-  let icon = 'fa-building';
+  // Pseudo style used to maintain spacing by hiding the assigned icon.
+  let icon = 'fa-blank';
 
   if (isPhone) icon = 'fa-phone-alt';
   if (isVideo) icon = 'fa-video';
```

Comment:
Based on Ciera's design guide, at home video appt and GFE device video appt will use the camera icon. ATLAS and video appt at VA clinic will use the bulding icon.

Reply:
Done